jvm/src/main/scala/PlatformReadInstances.scala

```diff
@@ -0,0 +1,39 @@
+package scopt
+
+import java.net.UnknownHostException
+
+private[scopt] object platform {
+  val _NL = System.getProperty("line.separator")
+
+  import java.util.{Locale, Calendar, GregorianCalendar}
+  import java.text.SimpleDateFormat
+  import java.io.File
+  import java.net.InetAddress
+
+  type ParseException = java.text.ParseException
+  def mkParseEx(s: String, p: Int) = new java.text.ParseException(s, p)
+
+  trait Implicits {
```

Comment:
I meant renaming `Implicits` trait to `PlatformReadInstances` trait, not the file name.
Sorry if I wasn't clear.